package.json

```diff
@@ -38,14 +38,12 @@
     "start:uhc-portal": "yarn --cwd ../uhc-portal start",
     "sync-dist": "./scripts/sync-dist.sh",
     "prepare": "install-peers",
-    "test": "run-s test:circular test:lint test:type-check",
-    "test:type-check": "tsc --noEmit",
-    "test:lint": "eslint --ext .js,.jsx,.ts,.tsx --color --max-warnings=0 ./src",
-    "test:unit": "cross-env CI=1 react-scripts test --env=jsdom",
+    "test": "run-s test:circular lint",
     "test:circular": "dpdm --warning false --tree false --exit-code circular:1 src/index.ts",
-    "test:watch": "react-scripts test --env=jsdom",
-    "lint": "eslint --ext .js,.jsx,.ts,.tsx --color --fix ./src",
+    "lint": "eslint ./src --ext .js,.jsx,.ts,.tsx --color",
+    "format": "yarn run lint --fix",
     "prettier": "prettier --write '**/*.{js,jsx,ts,tsx,json,md,scss,yaml,yml}'",
```

Comment:
Should we remove this in favor of `format` script above?

Reply:
The reason I separated them is that in CI you shouldn't execute `eslint ... --fix`
Also, please notice I dropped the `--max-warnings=0` flag.
There are 2 reasons for this: 
1. In practice, we ended up disabling rules in order to silence the linter instead of dealing with the problematic code.
2. Now we'll see the warnings and we can work iteratively in order to resolve them.